src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -51,7 +50,7 @@ private HearingsDetailsMapping() {
 
     }
 
-    public static HearingDetails buildHearingDetails(HearingWrapper wrapper, ReferenceDataServiceHolder referenceDataServiceHolder) {
+    public static HearingDetails buildHearingDetails(HearingWrapper wrapper, ReferenceDataServiceHolder referenceDataServiceHolder) throws URISyntaxException, IOException {
```

Comment:
Do we need the checked exceptions?

Reply:
Yes, if using resource loader, these checked exceptions need to be handled. I could handle them in ResourceLoader class itself, but in the application exceptions are thrown and propagated to the controllers rather than being handled at inception point, so followed same approach. Shall I handle it in ResourceLoader ? This depend on the review question above, if at all we need to use ResourceLoader

Reply:
I agree with @amittal89 here: if exceptions are propagated everywhere else, then we should keep the approach consistent.

A possible solution in other situations would be to use Project Lombok and the @SneakyThrows annotation but I don't know that that's applicable here and definitely not within the scope of this set of changes.

Reply:
Got rid of one of the checked exception after removing resourceLoader. If  we are able to inject the Resource, we would be able to get rid all cheked exceptions.

Reply:
As per comment on ResourceLoader: we can't inject the resource at present stage so this is good enough.

This improvement will have to go up as technical debt.